generators/openshift/templates/_deployment.yml

```diff
@@ -63,6 +57,12 @@ parameters:
     description: Port of the application
     required: true
     displayName: Application Port
+  -
+    name: NAMESPACE
+    value: "<%= openshiftNamespace %>"
+    description: Openshift Namespace
+    required: true
+    displayName: OpenShift namespace
```

Comment:
Templates are generated based on response to NAMESPACE prompt. I don't really see a value to parameterise this unless if you have a specific reason.

Reply:
Oh yeah, you're right.